stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -150,6 +150,11 @@ end
     r
 end
 
+function axes(D::Diagonal)
+    ax = axes(D.diag,1)
```

Comment:
```suggestion
    ax = axes(D.diag, 1)
```
Or a one-liner, similar to `size`
```suggestion
axes(D::Diagonal) = (ax = axes(D.diag, 1); (ax, ax))
```
?

Reply:
I've updated to use the one-line form